doc/devel/pr_guide.rst

```diff
@@ -205,9 +211,11 @@ Merging
     A core dev should only champion one PR at a time and we should try to keep
     the flow of championed PRs reasonable.
 
-* Do not self merge, except for 'small' patches to un-break the CI or
-  when another reviewer explicitly allows it (ex, "Approve modulo CI
-  passing, may self merge when green").
+After giving the last required approval, the author of the approval should
+merge the PR. PR authors must not self-merge, except for when another reviewer
+explicitly allows it (e.g., "Approve modulo CI passing, may self merge when
+green", or "Take or leave the comments. You may self merge".).
+Core developers may also self-merge in exceptional emergency situations.
```

Comment:
```suggestion
After giving the last required approval, the author of the approval should
merge the PR. PR authors should not self-merge except for when another reviewer
explicitly allows it (e.g., "Approve modulo CI passing, may self merge when
green", or "Take or leave the comments. You may self merge".).
```

I think changing the must -> should and dropping the last sentence side-steps the word choice issues.